### ISSUE TITLE
[Suggestion] Make a new server with a focus on realism and higher difficulty

### ISSUE THREAD

**Issue by alekb1** [EarthMC/Issue-Tracker#840]
This is a suggestion to add a fourth server to the list of servers on EMC. The original appeal for many of the players when they chose to join EMC was the realism and roleplaying aspect of an earth like Minecraft server. Adding to this, many veterans go inactive over time, as the challenge of Minecraft simply disappears on the server. This is why I'm suggesting making a way more challenging version of EMC, focusing on over world infrastructure, unexplored mechanics of Minecraft, colonial trade, and most important of all a harder difficulty for more experienced players. I believe that this is possible with the following 13 changes to the standard format of Terra Nova (TN):

1) /n spawn Is disabled, /res spawn is definitely disabled, and ideally even /t spawn. Currently it is almost a disadvantage to have public infrastructure in your nation, as half the people who use, are only using it to attack someone in the wild. This change would flip the initiative completely, as it would now be necessary for the individual towns to connect to a larger network in order to have access to mainstream trade. This will also allow for people to just have a city state. I have lead single town nations before, as i did not want to associate with the nation nearby, but still needed access to teleportation, as otherwise i would isolate myself. In my opinion, you should be able to be an independent town.
2) Disable passive healing. Now you have to carry medicine (regain pots and g. apples) with you. You’ll be able to heal in the towns with doctors (beacons). This is also to nerf absorption potions a little bit, as these don’t heal you. This will also encourage transport infrastructure to nearby larger cities that can afford beacons.
3) Packed ice and blue ice is expensive again. Ice is already significantly faster than any other alternative. Which I believe is a good thing. But rather than having it cheap, it should be something that took a lot of time to gather, as otherwise alternatives like horses and minecarts are not viable options.
4) All MCMMO abilities are enabled again, with the exception of the ones that can be used to get a lot of gold. This is just to add some of the old feel from classic, that a lot of the veteran community miss. PvP is painfully slow as it is, so boosting the damage output of swords and axes would not be a bad idea. This would also add bows as a viable option again, as bows are almost never used on the battlefield anymore, which is a real shame.
5) It costs 64 gold to create a town, and 64 gold to expand it, and the amount of claims are halved to 4 per citizen and a maximum of 70 extra from a nation. You have this tendency on TN (and classic) where people will claim as much area as possible. This results in giant areas claimed, where barely half of the town is actually developed. No one needs to be able to claim 940 chunks for 100 citizens, or just 220 chunks for 10, and it’s generally really disruptive for infrastructural projects when they have to coordinate and navigate around claims of many large towns. Hot spots on TN is already after 1 year of playtime getting very clustered, and I believe that the server would benefit greatly by just restricting crazy growth a little bit more. 
6) Nations now cost 2048 gold (or 2304, a full inventory of golden ingots). 1024 gold is honestly not a lot, for players after getting infrastructure setup. I know multiple people that have been able to make 500+ a day, and even I have tried making 1.7k in a full day of grinding. Creating a nation should be a public project, which is why it should also be difficult to achieve.
7) Adding a horse stat plugin like (https://dev.bukkit.org/projects/horsestats , maybe with anarchy mode). Horses is a insanely interesting, but unused mechanic of Minecraft, and have insane depth to it. But horses in TN are not able to be teleported, and are generally to slow be used, if they are not breed. The lack of teleportation, would entail a lot more off road traveling, which is where horses and mules shine. This revitalized change, would mean that horses would be valuable assets for traders, scouts, and maybe even soldiers. To encourage interests in horse breeding, I would suggest adding a plugin where people could tell the stats of a horse easily, as the traditional redstone method is way to unreliable with a (sometimes) unstable tick rate on the server.
8) Adding some minecart link plugin, for the ability to create trains for large scale item transport. Trains would be the logical way to transport large amounts of raw materials from one production site to a trade hub in a capital. For it to be a viable mode of transportation though, you’ll have to be able to link the minecarts as otherwise the janky physics of Minecraft will make it too inconsistent. Encouraging large train transports, will at the same time encourage events such as train raiding. With breed horses, you can move faster than minecarts. If you knew the expected leaving time of something like a supply train, you would be able to gather up a crew, and attack the train on horseback like in a old western movie.
9) MCMMO stats carry over. This way it will be less of a commitment for some people to join, as they don’t have to grind all over again (which veterans have already done twice).
10) End is still disabled. Elytra are to OP. This is one change I really appreciate in terra nova.
11) Disable end chest, but enable super expensive shulker boxes (like 9 gold blocks in a crafting grid, or maybe wither skull surrounded by gold ingots). Shulker boxes and enderchests are essentially the same thing, with the exception, that you can hide all your valuables in an enderchest at all time. If large trade routes become a thing, raiders of such routes should also be able to retrieve all the items that were being transported. Shulker boxes will also be a way to transport goods more efficiently over large distances, but at the same time risking a lot more, as they will be really expensive
12) (see attached picture for a rough sketch of what a potential ore gen could look like) Alter the ore generation, so that 80-90% of the ores on one continent is a certain type. To clarify, all ores will still be available, just in small trace amounts. Even on TN most people don’t care about the nations on other continents, but historically intercontinental trade has been a great business. But as it is now, the only trade that is done over long distance is the sand and ice trade. I have made a sketch of how I would distribute the ores over different continents. I would still have gold be equally accessible everywhere, as one continent with most of the gold would be overpowered. This idea is to stop natural resources from having a global price. If you live in an emerald rich area, emeralds are not gonna be worth a lot, but if there is barely any redstone around, suddenly the redstone is in very high demand. This will encourage intercontinental trade, which was a milestone in the history of the world. This would spark great trading companies like the East India Company, which made a fortune transporting items over large distances. By separating ores over such great distances, there will also be an industry of pirates, that will attack people crossing large bodies of water or land. 
13) Ideally also disable the nether, which would completely remove spawners from the game, and would force overworld transportation. Wither skeletons should though still be accessible in some way, as otherwise there would be no beacons, and the grind on this server would be a bit to much.

<img width="1427" alt="oregen" src="https://user-images.githubusercontent.com/47500274/69859151-90cae380-12ce-11ea-83fc-cada219e316b.png">
)

**Comment by DeadGlamour** [comment]
First of all, I just want to say I really like your idea in general, and impressed by the amount of writing, so let me just give some notes:
1.100% agree.
2. Not 100% sure if its a good idea, potions of healing and gapples are pretty rare, so people who just started playing might have a very hard time, maybe just drastically slowing the healing speed would be enough, also, you will take away 90% of the purpose of food (theres still villagers I suppose).
3. At start I thought it would be bad since moving all the way would take a ton of time as well, but I realize that specialization is a core part of this idea, which I support.
4. 100% agree.
5. You just made gold less vaulable by making everything else more specialized to geography and now you make it way too valuable again, I really like how it solves the problem in TN, in which gold is way too valuable, so it would prob be better to just keep the expansion price the same.
6. 1000% agree.
7. Seems like another useless plugin which wont really be used much.
8.Flying machines could do it but better, giving you cover from bows and raiders, just make a normal flying machine, add a minecart for yourself and stack storage minecarts, would be way better than normal minecarts.
9. Yeah Id love that, I wouldnt want to lose my grind.
10. I dont really care about the end anyway.
11. While I do support disabling ender chests, I dont think the admins want shulker boxes in the server, prob due to duping.
12. I really like this idea, but why stopping at ores? Why not making most to all farms unique to biomes, wheat wont grow in some biomes, like the biomes in russia and antartica, but grow everywhere else, cactus will only grow in deserts and maybe savanas, and so on, there are many more crops and items to be farmed, like coral, which is already locked to certain areas, tress could have that limitation as well, this could really add another level of trade.
13. So the only way to get massive amounts of xp now are furnaces? maybe there could be another way to get spawners?

**Comment by alekb1** [comment]
Thanks for the feedback, the ideal would obviously be to restrict growth of certain crops to certain climate belts, and would imo be a really cool mechanic. The reason I have only suggested the ore gen to be regional was to make it more simple, but the whole dynamic with import/export of not only ores, but also farmable goods would really add a special profile to every single region of emc.

**Comment by Aitonater** [comment]
For the pure fact emc is stretched as it is. I wouldn't want another server which drains time and resources from the main towny server.

**Comment by alekb1** [comment]
I would say fair point, but my point would be that we already have queues on weekend, where this would allow for more growth of the overall community.

Secondly for the sake of realism and challenge for the towns, I would enable pvp everywhere, including inside cities

**Comment by alekb1** [comment]
Or maybe not as it would make trading harder

**Comment by alekb1** [comment]
Maybe this combined with #928 for a more realistic and war intensive server, for more experienced players? That way, normal terra nova, could be for the part of the community that likes a sandbox feel, and just want to make town.

**Comment by dext3r9** [comment]
Would be nice to have the option of a more challenging version of Terra Nova

**Comment by alekb1** [comment]
Eagle senpai please notice this suggestion <3

**Comment by alekb1** [comment]
With all the people asking for a reset, I'm gonna bumb this idea again. some of the elements from this could satisfy a lot of people looking for a new experience, while those are not done with terra nova can continue on it.

**Comment by Crevel1** [comment]
Overall, I think that the concept of a 'Hardcore' EarthMC Server is great. It adds further incentive to the players who want a real challenge, further expanding the community. However, I do believe that the upkeep required to keep the entire network running smoothly is just too great for the Admins to handle. The network is constantly in maintenance, now imagine the network being in even more constant maintenance and the lack of focus on the other servers that undergo issues because of the amount of work needed to adjust balances on this new one. It could be done but it would most likely have to be at the expense of Factions, which I'm sure everyone would be fine with.

I'd like to reference point 7 for a moment; I think adding a plugin that can display Horse Stats is a fantastic idea. I used to play, and still continue to play a large serious roleplay with that feature. The feature has found itself to be extraordinarily useful in travelling large distances and has stimulated an industry revolved around breeding and selling top-tier horses that can traverse the large map. With a map as big as Terra Nova's, it would not only be ideal but essential that a plugin for Horse Stats was added.

tl;dr remove factions, add the proposed server with care and planning; also add Horse Stats
